config-templates/shopify/inventory_level.yaml

```diff
@@ -0,0 +1,29 @@
+data_source:
+  name: shopify_inventory_level
+  owners:
+    - support@transformdata.io
+  sql_table: SHOPIFY.INVENTORY_LEVEL ### Update to your opportunities schema.table
+  mutability:
+    type: immutable
+  identifiers:
+    - name: inventory_item_id
+      type: primary
+    - name: location_id
```

Comment:
there's no location datasouce with a primary location_id - should that be in here, too?